phc-rails/app/controllers/api/v1/accounts_controller.rb

```diff
@@ -2,15 +2,21 @@ class Api::V1::AccountsController < ApplicationController
   respond_to :json
   skip_before_action :verify_authenticity_token
 
+  def show
+    sf_id = params[:sf_id]
+    respond_with Account.find_by sf_id: sf_id
+  end
+
   def search
+    # ActiveRecord::Base.connection.execute("SELECT set_limit(0.001);")
     auth_token = request.headers["HTTP_AUTH_TOKEN"]
     user_id = request.headers["HTTP_USER_ID"]
     if auth_token && user_id
       if user_authenticated?(user_id, auth_token)
         first_name = request.headers["HTTP_FIRSTNAME"]
         last_name = request.headers["HTTP_LASTNAME"]
-        result = PersonAccount.fuzzy_search({first_name: first_name}, false).fuzzy_search({last_name: last_name}, false)
-        respond_with result
+        result = Account.fuzzy_search({:FirstName=>first_name, :LastName=>last_name}, false)
```

Comment:
Line is too long. [92/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.